src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.cs

```diff
@@ -40,6 +40,47 @@ public void TestSimpleNewVm()
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestSimpleNewVmWithSystemAssignedIdentity()
+        {
+            ComputeTestController.NewInstance.RunPsTest("Test-SimpleNewVmSystemAssignedIdentity");
+        }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestSimpleNewVmWithUserAssignedIdentity()
+        {
+            /**
+             * To record this test run these commands first :
+             * New-AzureRmResourceGroup -Name UAITG123456 -Location 'Central US'
+             * New-AzureRmUserAssignedIdentity -ResourceGroupName  UAITG123456 -NameUAITG123456Identity
+             * 
+             * Now get the identity :
+             * 
+             * Get-AzureRmUserAssignedIdentity -ResourceGroupName UAITG123456 -Name UAITG123456Identity
+             * Nore down the Id and use it in the PS code
+             * */
```

Comment:
Can you add these to your test script?

Reply:
These are already present in the script, unfortunately I can not call them inline from the script as the UserAssignedIdentity modules and feature are not in GA... that's why as of now the script requires certain steps to be performed manually in the record mode.